src/test/java/model/FruitTest.java

```diff
@@ -0,0 +1,29 @@
+package model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitTest {
+    private Fruit fruit;
+
+    @BeforeEach
+    void setUp() {
+        fruit = new Fruit("banana", 125);
+    }
+
+    @Test
+    void getName_ok() {
+        Fruit expected = new Fruit("banana", 125);
```

Comment:
you don't need to test models, they are tested indirectly when you are testing your services (you use models there)